sink-connector-lightweight/docker/clickhouse-sink-connector-lt-service.yml

```diff
@@ -2,7 +2,7 @@ version: "3.4"
 
 services:
   clickhouse-sink-connector-lt:
-    image: ${CLICKHOUSE_SINK_CONNECTOR_LT_IMAGE}
+    image: registry.gitlab.com/altinity-public/container-images/clickhouse_debezium_embedded:${CLICKHOUSE_SINK_CONNECTOR_LT_IMAGE}
```

Comment:
We have moved away from gitlab to dockerhub, https://hub.docker.com/layers/altinityinfra/clickhouse-sink-connector/408-97b1d3d83ef93c1b76a2b1c4d9c544dc67fbbec3-lt/images/sha256-d134bc05e50df7f63025e776ab6e3216c6622cd159eb0f2d459ea2ce8975f396?context=explore

Reply:
@subkanthi then it needs to be updated here too right? As I took reference from here.
https://github.com/Altinity/clickhouse-sink-connector/blob/develop/sink-connector-lightweight/docker/tag-push-latest-image.sh

Reply:
Also seems like the latest tag is not available in the docker hub currently.
```
image: altinityinfra/clickhouse-sink-connector:${CLICKHOUSE_SINK_CONNECTOR_LT_IMAGE}
```

Reply:
@subkanthi @Selfeer Can you confirm once if it's this one or other one?
- https://hub.docker.com/r/altinity/clickhouse-sink-connector/tags
- https://hub.docker.com/r/altinityinfra/clickhouse-sink-connector/tags

Reply:
@Selfeer pushed in changes to use the altinityinfra tags, please pull latest from develop  @AjinkyaTaranekar